eksrollup/lib/k8s.py

```diff
@@ -6,7 +6,7 @@
 import sys
 from .logger import logger
 from eksrollup.config import app_config
-
+import threading
```

Comment:
I am using threading as opposed to futures. Is there a difference? Would one be better than the other?

Reply:
They run the same way, futures make it easier to extract some value that the asynchronous process produces and/or another process that needs to trigger on its completion. If you just want a thread to run in the background and do stuff, threading is perfectly fine.

Reply:
Cool, I think for now threading fits our needs. Thanks, Mike!